edb/schema/types.py

```diff
@@ -104,6 +104,13 @@ class Type(
         ExprType,
         default=None, compcoef=0.909)
 
+    # True for views.  This should always match the value of
+    # `bool(expr_type)`, but can be exported in the introspection
+    # schema without revealing weird internals.
+    from_alias = so.SchemaField(
+        bool,
+        default=False, compcoef=0.909)
```

Comment:
Since this is a derivative of `expr_type` should we make `compcoef=1`?

Reply:
Yeah, sure, that makes more sense.

(tbh I'm not sure that the compcoef of `expr_type` makes sense anyway?)